src/main/scala/tests/objectSignatures.scala

```diff
@@ -2,6 +2,10 @@ package tests
 package objectSignatures
 
 class A[T]
+{
+    val a: String = "asd"
+    def method3() = "asd"
+}
 
 object A
```

Comment:
Can we add some membters here to tests if we render doc for it?

Reply:
Added some tests